src/components/Delegate/StakingOpportunities.tsx

```diff
@@ -133,7 +128,7 @@ export const StakingOpportunities = ({ assetId }: StakingOpportunitiesProps) =>
         Header: <Text translation='defi.rewards' />,
         accessor: 'rewards',
         display: { base: 'table-cell' },
-        Cell: ({ value }: { value: string }) => {
+        Cell: ({ value }: { value: string; row: { original: ActiveStakingOpportunity } }) => {
```

Comment:
Since we don't use `row`, I wouldn't go as far as typing it here, it's already typed from `data`

Reply:
Fixed!